File: dispatcher-app/src/app/modules/currencyRates/models/national-bank-currency-rate.ts
export class NationalBankCurrencyRate {
	constructor(private currencyRates: Partial<NationalBankCurrencyRate>) {
		this.Date = currencyRates.Date;
		this.Cur_Abbreviation = currencyRates.Cur_Abbreviation;
		this.Cur_Scale = currencyRates.Cur_Scale;
		this.Cur_Name = currencyRates.Cur_Name;
		this.Cur_OfficialRate = currencyRates.Cur_OfficialRate;
	}

	Cur_ID?: number;
	Date?: Date;
	Cur_Abbreviation?: string;
	Cur_Scale?: number;
	Cur_Name?: string;
	Cur_OfficialRate?: number;
}

File: Api/HomeBudget-Rates-Api/HomeBudget.Components.CurrencyRates/Models/Api/NationalBankCurrencyRate.cs
using System;

using Newtonsoft.Json;

namespace HomeBudget.Components.CurrencyRates.Models.Api
{
    public class NationalBankCurrencyRate
    {
        [JsonProperty(PropertyName = "Cur_ID")]
        public int CurrencyId { get; set; }

        [JsonProperty(PropertyName = "Date")]
        public DateTime UpdateDate { get; set; }

        [JsonProperty(PropertyName = "Cur_Abbreviation")]
        public string Abbreviation { get; set; }

        [JsonProperty(PropertyName = "Cur_Scale")]
        public int Scale { get; set; }

        [JsonProperty(PropertyName = "Cur_Name")]
        public string Name { get; set; }

        [JsonProperty(PropertyName = "Cur_OfficialRate")]
        public decimal? OfficialRate { get; set; }
    }
}
